samcli/commands/deploy/exceptions.py

```diff
@@ -67,3 +67,12 @@ def __init__(self):
         )
 
         super(DeployBucketRequiredError, self).__init__(message=message_fmt)
+
+
+class DeployResolveS3AndS3Error(UserException):
+    def __init__(self):
+        message_fmt = (
+            "Cannot use both --deploy-s3 and guided or --s3-bucket parameter simultaneously." "Please use either one."
```

Comment:
Will want to update this error message to reflect that guided is not part of the conflict.